_includes/code/components/sign-in-form.html

```diff
@@ -1,3 +1,3 @@
 {% capture sign-in-form %}
-  {% fractal_component sign-in-form %}
```

Comment:
Seems like the [sign in form pattern](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/al-beta-integration/templates/form-templates/sign-in-form/) attributes aren't coming through.

https://user-images.githubusercontent.com/3385219/163457181-4878fc5e-f2fd-4068-9218-c7e36ab58cf6.mp4